cmd/minikube/cmd/start.go

```diff
@@ -357,6 +358,18 @@ func validateConfig() {
 			exit.Usage("Sorry, the kubeadm.%s parameter is currently not supported by --extra-config", param)
 		}
 	}
+
+	// This validates if the --registry-mirror args
+	// match the format of http://localhost
+	urlRe := regexp.MustCompile(`^http(s{0,1}):\/\/(.+?)`)
```

Comment:
This looks roughly correct, but I don't believe that directories are allowed in the registry path

Rather than maintaining our own regular expression, I'd prefer this to re-use the parser provided by the URL library: url.Parse(). It's a bit stricter too. 

Once you have a URL object, you can check that the Scheme is as expected, and that the Path is empty.

Reply:
yeah, url.parse is cleaner. Thank you!

Reply:
thanks